src/GT.Data/Repositories/Impl/GTGenericRepository.cs

```diff
@@ -131,33 +131,26 @@ public async Task UpdateAsync(TEntity entity, string id)
 		/// </summary>
 		/// <param name="id">ID of the entity to be deleted.</param>
```

Comment:
Method summary should be edited to reflect changes, i.e. refer to the "entity" parameter rather than "id".